src/components/Card/PCardBase.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <div
+    class="p-card"
+    :class="normalizeClass([props.class, card({ variant })])"
```

Comment:
again, don't make a class prop. But also if you have an array of classes, just bind them directly to the class attribute, vue will handle the normalizing for you. So you don't need to use normalizeClass here.